framework/uicomponents/qml/MuseScore/UiComponents/CheckBox.qml

```diff
@@ -74,7 +85,7 @@ FocusableItem {
                 target: box
                 radius: 1
                 color: Qt.rgba(ui.theme.button.r, ui.theme.button.g, ui.theme.button.b, 1.0)
-                border.color: Qt.rgba(0, 0, 0, 0.15)
+                border.color: globalStyle.highlight
```

Comment:
It is better to use `ui.theme.highlight`
(At the moment `ui.theme` is similar to `globalStyle`,  but in the future it is likely to change)

Reply:
done